apps/site/src/pages/api/types/entity-type/aggregate.api.ts

```diff
@@ -0,0 +1,31 @@
+import { EntityTypeWithMetadata } from "@blockprotocol/graph";
```

Comment:
The property type and entity type versions of these endpoints and db implementations is very similar. It could probably be deduplicated.